CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing
+
+We would absolutely love any pull requests to contribute materials to our RSS feed of curated materials about reproducibility. If you add resources to our feed, the contributions will be tweeted out via the [@ReproFeed](https://twitter.com/ReproFeed) twitter bot, and added to the directory on [reproduciblescience.org](https://reproduciblescience.org/directory).
+
+## Best Practices for Contributing
+* Make sure that the resource you are suggesting isn't already in the feed. 
+* Make sure the resource you are suggesting is open access. This means that anyone should be able to access, read, and/or download it without having to pay money or login.
```

Comment:
I like this requirement! The RSS definitely shouldn't be sending people to paywalls.